molecule/default/requirements.yml

```diff
@@ -3,10 +3,6 @@
 
 - src: onaio.mailroom
 
-- src: https://github.com/DavisRayM/python.git
-  name: DavisRayM.python
-  scm: git
-
```

Comment:
`onaio.django` now adds this as a dependency and specifies which `version`, `scm` and `source` to use.

_[Change that implemented this](https://github.com/onaio/ansible-django/blob/master/meta/main.yml#L14-L24)_